File: esc180/fall semester/assignments/lab9/sort.c
int bubbleSort(int *array, int size);

int bubbleSort(int *array, int size){
	int nLength;
	int swapped;

	nLength = size;
	swapped = 1;

	if(array == NULL){
		return -1;
	}

	while(swapped){
		int i;
		int temp;
		swapped = 0;
		for(i = 1; i < nLength; i++){
			if(array[i-1] > array[i]){
				temp = array[i];
				array[i] = array[i-1];
				array[i-1] = temp;
				swapped = 1;
			}	
		}
	}
	
	return 0;
}	
				

File: esc180/fall semester/assignments/lab9/sort.py
def bubbleSort(a):
	swapped = True
	array = list(a)
	n = len(array)

	try:
		if n < 1:
			return [False, array]
	
		while swapped:
			swapped = False
			for i in range(1,n):
				if array[i-1] > array[i]:
					temp = array[i-1]
					array[i-1] = array[i]
					array[i] = temp
					swapped = True
	
		return [True, array]
	except:
		return [False, array]

 				
